charts/nfdiv-case-api/Chart.yaml

```diff
@@ -3,10 +3,13 @@ appVersion: "1.0"
 description: A Helm chart for nfdiv-case-api App
 name: nfdiv-case-api
 home: https://github.com/hmcts/nfdiv-case-api
-version: 0.0.1
+version: 0.0.2
 maintainers:
   - name: HMCTS nfdiv team
 dependencies:
   - name: java
     version: 3.4.4
     repository: '@hmctspublic'
+  - name: ccd
+    version: ~4.3.2
```

Comment:
we will also need more chart config similar to this https://github.com/hmcts/nfdiv-ccd-definitions/blob/master/charts/nfdiv-ccd-definitions/values.preview.template.yaml